src/app/[locale]/(main)/quizbank/[id]/components/view-AI-explain.tsx

```diff
@@ -0,0 +1,25 @@
+import { PiSparkle } from "react-icons/pi"
+
+type Props = {
+  explain: string
+  classname: string
+}
+
+export default function ViewAIExplain({ explain, classname }: Props) {
+  return (
+    <div
+      className={`${classname} mt-5 whitespace-nowrap border-t-2 border-gray-300 leading-[150%] max-md:mb-10`}
```

Comment:
use `cn`, refer to button that use cn to merge props.className and own classname